### ISSUE TITLE
Series with "&" in the name can't be opened

### ISSUE THREAD

**Issue by CennoxX** [madnight/imdb-series-chart#37]
If I search for "Love, Death & Robots", "Trial & Error", ... I find the shows, but if I try to open the series, I get the message "TV Show not found!" When opening with the IMDb-ID there's no problem.

**Comment by madnight** [comment]
Yes, seems to be a bug with the "&" symbol. Have you tried to search for tt9561862 (imdb id)?
